src/desktop/chat/chatwidget.h

```diff
@@ -78,6 +78,13 @@ private slots:
 	void expandRequested();
 
 private:
+#ifdef Q_OS_ANDROID
```

Comment:
Why not just `#ifdef` out the code directly? Is there some plan for this to apply elsewhere later?

Reply:
The ifdefs just added a lot of clutter in headers, sources of both this file and the signal connections made to it, to the point where I got myself confused a few times. So it's just less of a mess, but it could just as well be ifdef'd out directly if you think that would make more sense.